src/connectors/infinite-hits/connectInfiniteHits.ts

```diff
@@ -158,7 +157,7 @@ const connectInfiniteHits: InfiniteHitsConnector = function connectInfiniteHits(
           ...helper.state,
           page: getFirstReceivedPage() - 1,
         })
-        .search();
+        .searchWithoutTriggeringOnStateChange();
```

Comment:
It searches without triggering `onStateChange` on InstantSearch instance, because this is meant to be silent and to not modify the state.